cmd/minishift/cmd/start.go

```diff
@@ -87,6 +82,13 @@ var minishiftToClusterUp = map[string]string{
 
 // init configures the command line options of this command
 func init() {
+	startCmd = &cobra.Command{
+		Use:   commandName,
+		Short: "Starts a local OpenShift cluster.",
+		Long:  `Starts a local single-node OpenShift cluster on the specified hypervisor.`,
```

Comment:
`on the specified hypervisor` -> `using the native hypervisor`. I think we specify when we need.